### DISCUSSION TITLE
Reflections on 7 years of P2G

### DISCUSSION THREAD

**Discussion by philosowaffle** [philosowaffle/peloton-to-garmin#599] [General]
Over the years I've built many tools and weekend scripts, but [Peloton-to-Garmin](https://github.com/philosowaffle/peloton-to-garmin?ref=next-topic.com) is the first tool I wrote that was used by a community of people and it's been so much fun growing and learning how to maintain a free open-source project.

The new year caused me to reflect a bit and realize its been 7(!) years now of working on this project.  I decided to write up a longer form post on some of the history of P2G, mostly for my own benefit of remembering how this project started and has grown over time. You can find it [here](https://www.next-topic.com/reflections-on-7-years-of-p2g/).

**Comment by jkramerilegv** [comment]
PW, thanks again for all your work on this project.  My wife and I use it every day and it is a godsend.  I will continue to buy you coffee as it is the least I could do for using and abusing one of the most valuable tools in my lifestyle (with my Sawzall a close second :) ).  Your 7 Year reflections document further escalates my appreciation for the task you took on and knocked out of the park.

**Comment by gunjanmarwah** [comment]
Just wanted to say thanks for building this and for supporting it all these years. Now that Peloton and Garmin have finally released their own 2-way integration, I'm excited to finally disable my workflow here.
